data/transactions/verify/txn.go

```diff
@@ -156,19 +196,22 @@ func txnGroupBatchPrep(stxs []transactions.SignedTxn, contextHdr bookkeeping.Blo
 	}
 	feeNeeded, overflow := basics.OMul(groupCtx.consensusParams.MinTxnFee, minFeeCount)
 	if overflow {
-		err = fmt.Errorf("txgroup fee requirement overflow")
-		return
+		err = &ErrTxGroupInvalidFee{err: fmt.Errorf("txgroup fee requirement overflow")}
```

Comment:
This counter might not help much, and could be 0, even if the transactions are being dropped because of the fee.
Many of the end points already make sure the transaction has sufficient fees, so maybe it is unlikely this will hit?
More interesting failure of a transaction happens later, when either the fee is not sufficient because the pool fee has gone up (full blocks), or when the account does not have the fee to cover for the transaction.

Reply:
let's see if it pops ups, and remove if not useful